### ISSUE TITLE
Non-Merge Access Users Cannot View Specific Merge Conflicts in Pull Requests They Review

### ISSUE THREAD

**Issue by monilpat** [elizaOS/eliza#457]
**Describe the bug**

Users without merge access are unable to view specific merge conflicts in pull requests. This limitation prevents them from resolving or contributing to conflict resolution effectively, as they cannot see the conflicting files or lines of code.

**To Reproduce**

1. Create a pull request with merge conflicts.
2. Ensure that the user reviewing the PR does not have merge access.
3. Attempt to view the specific merge conflicts as the non-merge access user.

**Expected behavior**

Users without merge access should be able to view the specific merge conflicts within the pull request, including the files and lines of code in conflict, to assist in resolving the issue.

**Screenshots**

**Additional context**

This limitation may hinder collaboration and delay the resolution of merge conflicts, especially in teams where code reviews and conflict resolutions involve contributors without merge privileges.

**Comment by sirkitree** [comment]
Any suggestions on how to fix this? Is there a permission or something in gh we can update?

**Comment by monilpat** [comment]
Yeah for sure there has to be some settings in the actions yaml files that configures this, will have a look, just wanted to flag that either way a maintainer will need to address this! Thanks so much!
